core/src/main/java/de/neofonie/styla/core/utils/SeoUtils.java

```diff
@@ -15,24 +13,35 @@
 
 public class SeoUtils {
 
+    private static final String TAG_LINK = "link";
+    private static final String TAG_META = "meta";
+
+    private static final String ATTRIBUTE_REL = "rel";

```

Comment:
I guess the . TITLE is set differently

Reply:
The title is currently set by the page itself which you create in aem ...